docs/developing.rst

```diff
@@ -0,0 +1,155 @@
+Developer Guidelines
+====================
+
+Git Workflow
```

Comment:
I really think this should go back to the CONTRIBUTING...

Reply:
The `git` workflow or the whole page?